sdk/core/scheduler/main.cc

```diff
@@ -170,243 +170,235 @@ namespace
 
 } // namespace
 
-namespace sched
```

Comment:
These things should all be moved to an anonymous namespace, not exported functions in the global namespace.

Ideally, the `priv` namespace should also be removed.

Reply:
OK, I've done the first thing. Had to move exception_entry and scheduler_entry out of the anonymous namespace. The `priv` namespace should be a separate PR, I think.

Reply:
I don't think you pushed?  It is still putting everything in the global namespace.  Making it an anonymous namespace should remove most of the whitespace changes.